package.json

```diff
@@ -27,7 +27,8 @@
     "fhirclient": "^0.1.10",
     "jquery": "^3.1.1",
     "react": "^15.4.1",
-    "react-dom": "^15.4.1"
+    "react-dom": "^15.4.1",
+    "react-intl": "^2.2.3"
```

Comment:
can you use the NPM package for fhir client?
https://www.npmjs.com/package/fhirclient

Reply:
Unfortunately, there's no module to import with this library. The npm package is just for convenience so a consumer can reference the actual `fhirclient` JS file in `node_modules`. Forgot to take the `fhirclient` dependency out of here, once I figured that out. 

See [here](https://groups.google.com/forum/#!topic/smart-on-fhir/UIa2XNqUNqI)

Reply:
That is pretty useless. :(